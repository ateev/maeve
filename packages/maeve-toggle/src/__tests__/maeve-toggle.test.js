import React from 'react';
import { shallow } from 'enzyme';
import MaeveToggle from '../../index.js';

test('render ', () => {
  const component = shallow(
    <MaeveToggle />
  );
  expect(component).toMatchSnapshot();
});