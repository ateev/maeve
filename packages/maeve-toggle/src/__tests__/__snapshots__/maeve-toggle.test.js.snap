exports[`test render  1`] = `
<div
  className="maeve-toggle">
  <label />
  <input
    className="toggle-input"
    type="checkbox" />
  <label>
    <span
      className="label-inner" />
    <span
      className="label-switch" />
  </label>
</div>
`;
